docs/docs/getting-started.mdx

```diff
@@ -798,6 +802,26 @@ MDX files can be imported in Node by using
 [`@mdx-js/node-loader`][mdx-node-loader].
 See its readme on how to configure it.
 
+#### Bun
+
+MDX files can be imported in [Bun][] by using
+[`@mdx-js/esbuild`][mdx-esbuild].
+
+<details>
+  <summary>Expand example</summary>
+
+  ```toml path="bunfig.toml"
+  preload = ["./bunMdxEsbuild.ts"]
+  ```
+
+  ```js twoslash path="bunMdxEsbuild.ts"
```

Comment:
One thing I wonder about is this casing in the filename? Is this a very strong recommendation by bun, to use camelcasing? Seems like that would fail on some file systems?

Should the name include esbuild, or is that a side effect of what the focus is: Bun + MDX?

How about a short:

````suggestion
  preload = ["./mdx.ts"]
  ```

  ```js twoslash path="mdx.ts"
````

Reply:
`bun-preload.ts` also seems viable?

Reply:
> Is this a very strong recommendation by bun, to use camelcasing?

It was my own choice. Camel case in filenames are pretty common in JavaScript and TypeScript ecosystems.

I think `mdx.ts` is not expressive enough. If it should be lowercase (kebab case?) then at least `bun-mdx.ts` or `bun-mdx-plugin.ts`

Reply:
OK, I’m very :-1: on camelcase in filenames, I only indeed sometimes see React people use it more recently. It’s not used anywhere in the things I work on.

Did you see my 2nd suggestion, `bun-preload.ts`?

My acceptable preferences in order are: `bun-preload.ts`, `mdx.ts`, `bun.ts`, `bun-mdx.ts`. All fine. They all answer “wait, why is this file here again?”.

I think the inclusion of “plugin” in `bun-mdx-plugin.ts` is unneeded. Similar to “esbuild” `bunMdxEsbuild.ts`. These terms have to do with how that file currently works internally.

Reply:
Ok, switched to `bun-mdx.ts` in 3de41ee65c2527ae1f91aaf0c220ce97cc6f0fdf